pages/recipes/supabase.mdx

```diff
@@ -224,7 +224,7 @@ With all the above steps complete, we can now restore!
 
 1. Run `snaplet snapshot restore --no-reset` in your project path.
 
-> **Note on `--no-reset`**: This will skip the “reset” step of the restore command. So no existing schemas will be dropped. Learn more about data operations [here](https://docs.snaplet.dev/getting-started/restoring#more-granular-control-over-restorations)
+> **Note on `--no-reset`**: This will skip the “reset” step of the restore command. So no existing schemas will be dropped. Learn more about data operations [here](/core-concepts/capture)
```

Comment:
This was not a perfect match, but close enough. A lot of these guides are quite old and no longer super relevant, but I wanted all the links to be valid.